### DISCUSSION TITLE
[examples]Does the example no longer match the current vite？

### DISCUSSION THREAD

**Discussion by RSS1102** [unocss/unocss#2896] [Q&A]
I found that the examples are no longer consistent with the corresponding template files generated by the current vite, with significant differences, such as  `tsconfig.node.json `. 
Additionally, some examples exist for `uno. config.ts`, while others do not. Causing certain ideological confusion.

---
I would like to ask if updates should be kept consistent. At least the configuration of each file should be consistent as much as possible for vite.
![image](https://github.com/unocss/unocss/assets/81673017/dc804d92-5ffb-4bb8-8fe3-8dc030ea9a3f)
![image](https://github.com/unocss/unocss/assets/81673017/85b19f35-b86a-4611-a928-e5e5045cb06f)

**Comment by zyyv** [comment]
Yes, `uno.config.ts` is missing in the example, we put it into `vite.config.ts` for example in order to achieve a minimal project structure.
But we still recommend that you use a single-file configuration to get the best development experience, such as vscode autocomplete tips.
As for the starter project template, you can check the documentation to get started https://unocss.dev/guide/config-file
